core/TaskObserver.js

```diff
@@ -124,6 +132,11 @@ class CompoundObserver extends TaskObserver {
         return this._subtasks.some(t => t.isPending);
     }
 
+    @computed
+    get pendingTaskCount() {
```

Comment:
Don't you want the implementation of this to refer to the `pendingTaskCount` of the subTasks?